components/playground/main.go

```diff
@@ -296,7 +296,7 @@ If you'd like to use a TiDB version other than %s, cancel and retry with the fol
 	defaultOptions := &BootOptions{}
 
 	rootCmd.Flags().String(mode, defaultMode, "TiUP playground mode: 'tidb', 'tikv-slim'")
-	rootCmd.Flags().Bool(withMonitor, false, "Start prometheus and grafana component")
+	rootCmd.Flags().Bool(withMonitor, true, "Whether to start prometheus and grafana component, default value is `--monitor=true`")
```

Comment:
```suggestion
	rootCmd.Flags().Bool(withMonitor, true, "Whether to start prometheus and grafana component. (default: true)")
```

tend to be the same style of default message generated by `flags` and it is neat.

Reply:
maybe we need to tell users how to set it? actually, `tiup playground nightly --monitor false` is useless.

Reply:
In my opinion it is go flags responsibility. But it is not a strong requirement from my side. May @lucklove & @lilyjazz have other suggestion.